local-cli/runIOS/runIOS.js

```diff
@@ -20,82 +20,106 @@ const Promise = require('promise');
  * Starts the app on iOS simulator
  */
 function runIOS(argv, config) {
-  return new Promise((resolve, reject) => {
-    _runIOS(argv, config, resolve, reject);
-    resolve();
-  });
+  return _runIOS(argv, config);
 }
 
-function _runIOS(argv, config, resolve, reject) {
-  const args = parseCommandLine([
-    {
-      command: 'simulator',
-      description: 'Explicitly set simulator to use',
-      type: 'string',
-      required: false,
-      default: 'iPhone 6',
-    }, {
-      command: 'scheme',
-      description: 'Explicitly set Xcode scheme to use',
-      type: 'string',
-      required: false,
-    }, {
-      command: 'project-path',
-      description: 'Path relative to project root where the Xcode project (.xcodeproj) lives. The default is \'ios\'.',
-      type: 'string',
-      required: false,
-      default: 'ios',
+function _runIOS(argv, config,) {
+  return new Promise((resolve, reject) => {
```

Comment:
Could you keep the original function structure (creating the Promise in `runIOS`), as it is it's really hard to see what code actually changed.